### ISSUE TITLE
Text not showing up right after view switches

### ISSUE THREAD

**Issue by pvcraven** [pythonarcade/arcade#1073]
Run this for a while. The text eventually starts glitching:

```python
import arcade

class MyView(arcade.View):
    def __init__(self):
        super().__init__()
        arcade.set_background_color(arcade.color.BLACK)
        self.elapsed_time = 0.0
        self.text1 = arcade.Text(
            'text',
            20,
            20,
            font_size=60
        )

    def on_draw(self):
        arcade.start_render()
        self.text1.draw()

    def on_update(self, delta_time):
        self.elapsed_time += delta_time
        if self.elapsed_time >= 0.05:
            self.window.show_view(MyView())

if __name__ == '__main__':
    window = arcade.Window(400, 400, 'Bug')
    window.show_view(MyView())
    arcade.run()
```

**Comment by pvcraven** [comment]
![bug](https://user-images.githubusercontent.com/1463116/151567614-deace750-20a7-4f65-a52e-1118228bc9e7.gif)

**Comment by einarf** [comment]
I can reproduce this issue as well. Nice work findig this.

**Comment by einarf** [comment]
Should be fixed in `pyglet=2.0.a1`.

**Comment by einarf** [comment]
Finally this bug is squashed. 2.6.13 is released using a newer pyglet version.
